<launch>
    <node pkg="follow_the_leader" type="track_continously" name="track_continously" output="screen">
    <param name="max_n_track_before_det_count" type="int" value="30" />
    <param name="tracking_threshold" type="double" value=".8" />
    <param name="min_allowed_tracking_treshold" type="double" value=".6" />
    <param name="max_strike" type="int" value="10" />
    
    <!-- param name="mav_name" type="string" value="$(arg mav_name)" / -->
    </node>
    <comment launch-prefix="gdb -ex run --args"/>


    <node pkg="follow_the_leader" type="buffer_then_track" name="buffer_then_track" output="screen">
    <param name="frame_to_process_upper_bound" type="int" value="5"/> 
    <param name="max_n_track_before_det_count" type="int" value="30" />
    <param name="tracking_threshold" type="double" value=".8" />
    <param name="min_allowed_tracking_treshold" type="double" value=".6" />
    <param name="max_strike" type="int" value="10" />
    
    <!-- param name="mav_name" type="string" value="$(arg mav_name)" / -->
    </node>





</launch>
